corehq/apps/data_interfaces/views.py

```diff
@@ -99,14 +99,15 @@ class BulkUploadCasesException(Exception):
 
 class DataInterfaceSection(BaseDomainView):
     section_name = ugettext_noop("Data")
+    urlname = 'data_interfaces_default'
 
     @method_decorator(require_can_edit_data)
     def dispatch(self, request, *args, **kwargs):
         return super(DataInterfaceSection, self).dispatch(request, *args, **kwargs)
 
     @property
     def section_url(self):
-        return reverse("data_interfaces_default", args=[self.domain])
+        return reverse(self.urlname, args=[self.domain])
```

Comment:
@orangejenny it looks like this is causing https://sentry.io/dimagi/commcarehq/issues/657007603/?referrer=slack. it would also affect any other subclass of DataInterfaceSection whose url takes more args than just domain

Reply:
Thank you for finding! https://github.com/dimagi/commcare-hq/pull/21615